docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -79,6 +79,27 @@ Auth Service's gRPC endpoint.
 
 The Helm chart only supports the `file` format.
 
+```code
+$ tctl auth sign --user=access-plugin --out=auth
+```
+
+Then create a Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-mattermost-identity --from-file=auth_id=auth.pem
+```
+
+Teleport's Access Request plugins listen for new and updated Access Requests by
```

Comment:
For some reason, the paragraphs beginning "Teleport's Access Request plugins listen..." and "The identity file, `auth.pem`" are repeated in this `TabItem`, as are the `kubectl create secret` and `tctl auth sign` commands. We should clean this `TabItem` up before we merge.

Reply:
Oh yeah, the reason might've been that I needed to add the same changes to both PRs and it got auto-merged this way.